### ISSUE TITLE
High severity vuln with cross-spawn in npm package generated

### ISSUE THREAD

**Issue by mpizenberg** [axodotdev/cargo-dist#1583]
Thank you for dist! I’m using it to publish some rust project I have and it seems like the generated npm package has a version of `cross-spawn` marked with a high severity vuln. Do you know why it would pick that version (7.0.3) when a newer version without the vuln exists, and it doesn’t seem to be a problem for the constraints of the parent dependency?

```txt
❯ npm ls cross-spawn
elm-cardano@0.1.6 /Users/piz/Downloads/package
└─┬ rimraf@5.0.8
  └─┬ glob@10.3.16
    └─┬ foreground-child@3.1.1
      └── cross-spawn@7.0.3
```

**Comment by duckinator** [comment]
I'm not really familiar with this part of the codebase, but it looks like we're using [`npm shrinkwrap`](https://docs.npmjs.com/cli/v10/commands/npm-shrinkwrap), the results of which are [committed to the repo as `npm-shrinkwrap.json`](https://github.com/axodotdev/cargo-dist/blob/main/cargo-dist/templates/installer/npm/npm-shrinkwrap.json) and [read by `backend/installer/npm.rs`](https://github.com/axodotdev/cargo-dist/blob/main/cargo-dist/src/backend/installer/npm.rs#L43).

It looks like [rimraf was last updated in July](https://github.com/axodotdev/cargo-dist/pull/1202). Unfortunately, that means dependabot didn't get a chance to update `npm-shrinkwrap.json` for us, because [cross-spawn 7.0.4 through 7.0.6 were only released this month](https://www.npmjs.com/package/cross-spawn?activeTab=versions).

So in theory I think this should just be a matter of us updating `npm-shrinkwrap.json`.

**Comment by duckinator** [comment]
This is fixed by #1584.
